04_set_up_the_folio_users_app.md

```diff
@@ -148,7 +149,7 @@ $ mvn install
 ```
 
 ### Register and Deploy the Users app Okapi Module
-The Git repository for Okapi Module has a [Module Descriptor](https://github.com/folio-org/mod-users/blob/master/ModuleDescriptor.json) and a [Deployment Descriptor](https://github.com/folio-org/mod-users/blob/master/DeploymentDescriptor.json) that can be used to register and deploy the Users app Okapi Module.
+The Git repository for the Users app Okapi Module has a [Module Descriptor](https://github.com/folio-org/mod-users/blob/master/ModuleDescriptor.json) that can be used to register the Users app Okapi Module.
```

Comment:
I saw that there was a [pull request on folio-org/mod-users](https://github.com/folio-org/mod-users/pull/13) that removed the excess key-value pairs from the Deployment Descriptor.  Even with those the Deployment Descriptor worked.  Is there any reason not to use the one from mod-users (with our without the extra lines)?